app/routes/task_routes.py

```diff
@@ -1 +1,220 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, request, Response, jsonify
+from app.models.task import Task
+from ..db import db
+from datetime import datetime
+import os
+from dotenv import load_dotenv
+import requests
+from app.routes.route_utilities import validate_model, check_complete
+
+load_dotenv()
+SLACKBOT_TOKEN = os.getenv("SLACKBOT_TOKEN")
+
+tasks_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+
+@tasks_bp.post("")
+def create_task():
+
+    request_body = request.get_json()
+
+    if "title" not in request_body:
+        return {"details": "Invalid data"}, 400
+    
+    if "description" not in request_body:
+        return {"details": "Invalid data"}, 400
+
+    if "completed_at" in request_body:
+        completed_at = request_body["completed_at"]
+
+    else:
+        completed_at = None
+
+    new_task = Task(title=request_body["title"], description=request_body["description"], completed_at=completed_at )
+    db.session.add(new_task)
+    db.session.commit()
+
+    response = {
+        "task": {
+        "id": new_task.id,
+        "title": new_task.title,
+        "description": new_task.description,
+        "is_complete": check_complete(new_task.completed_at)
+
+    }
```

Comment:
By tweaking the to_dict for Task above, we could absolutely make this a method that creates the dictionary we are looking for without rewriting the dictionary like this.